spiped/dispatch.c

```diff
@@ -199,25 +200,25 @@ callback_gotconn(void * cookie, int s)
 }
 
 /**
- * dispatch_accept(s, tgt, rtime, sas, decr, nopfs, requirepfs, nokeepalive, K,
- *     nconn_max, timeo, conndone):
- * Start accepting connections on the socket ${s}.  Connect to the target
- * ${tgt}, re-resolving it every ${rtime} seconds if ${rtime} > 0; on address
- * resolution failure use the most recent successfully obtained addresses, or
- * the addresses ${sas}.  If ${decr} is 0, encrypt the outgoing connections; if
- * ${decr} is non-zero, decrypt the incoming connections.  Don't accept more
- * than ${nconn_max} connections.  If ${nopfs} is non-zero, don't use perfect
- * forward secrecy.  If ${requirepfs} is non-zero, require that both ends use
- * perfect forward secrecy.  Enable transport layer keep-alives (if applicable)
- * if and only if ${nokeepalive} is zero.  Drop connections if the handshake or
- * connecting to the target takes more than ${timeo} seconds.  If
+ * dispatch_accept(s, tgt, rtime, sas, sa_b, decr, nopfs, requirepfs, nokeepalive,
+ *     K, nconn_max, timeo, conndone):
+ * Start accepting connections on the socket ${s}, optionally binding to ${sa_b}.
+ * Connect to the target ${tgt}, re-resolving it every ${rtime} seconds if
+ * ${rtime} > 0; on address resolution failure use the most recent successfully
+ * obtained addresses, or  the addresses ${sas}.  If ${decr} is 0, encrypt the
+ * outgoing connections; if ${decr} is non-zero, decrypt the incoming connections.
+ * Don't accept more than ${nconn_max} connections.  If ${nopfs} is non-zero,
+ * don't use perfect forward secrecy.  If ${requirepfs} is non-zero, require that
+ * both ends use perfect forward secrecy.  Enable transport layer keep-alives
+ * (if applicable) if and only if ${nokeepalive} is zero.  Drop connections if the
+ * handshake or connecting to the target takes more than ${timeo} seconds.  If
  * dispatch_request_shutdown() is called then ${conndone} is set to a non-zero
  * value as soon as there are no active connections.  Return a cookie which can
  * be passed to dispatch_shutdown() and dispatch_request_shutdown().
  */
 void *
 dispatch_accept(int s, const char * tgt, double rtime, struct sock_addr ** sas,
-    int decr, int nopfs, int requirepfs, int nokeepalive,
+    const struct sock_addr * sa_b, int decr, int nopfs, int requirepfs, int nokeepalive,
```

Comment:
Unfortunately this line ends up being 96 characters long; however, the `STYLE` file in the top-level directory calls for lines to "generally be 78 characters, and not more than 80 characters".

Please move `const struct sock_addr *` to the next line, and then the `size_t` onto the line after that.

Please also rewrap the function comment, as some of those lines are move than 80 chars.